dts/bindings/mtd/eeprom-spi-i2c.yaml

```diff
@@ -23,4 +23,7 @@ properties:
     wp-gpios:
       type: phandle-array
       required: false
-      description: GPIO to which the write-protect pin of the chip is connected
+      description: |
+        GPIO to which the write-protect pin of the chip is connected.
+
+        The device will interpret this signal as active-low.
```

Comment:
I am unsure as to why this change was introduced? It is important that we do not impose and `active-low` logic on the WP binding.

While it is true that most if not all serial EEPROMs have an active-low WP pin, external circuitry can invert the signal between the SoC and the EEPROM. Take for instance EEPROMs connected through a multi-master I2C bus. Since the bus is multi-master, the WP pin is likely to be controlled by multiple masters as well. The requires external circuitry in order not to connect two output pins together. This circuit may or may not negate the signal along its path. We need to support both.

Reply:
That case should be covered.   The purpose of this description is to explain what the device normally expects, so people adding devicetree nodes know that the EEPROM itself expects active-low.  If there is external circuitry it's the responsibility of the hardware designer and devicetree author to select the correct active level for whatever's driven by the GPIO specified in this property.

In other words: this conversion supports both active low and active high control.